src/app.rs

```diff
@@ -2261,6 +2279,25 @@ impl eframe::App for ProfApp {
             ..
         } = self;
 
+        if cx.take_screenshot {
+            frame.request_screenshot();
+            cx.take_screenshot = false;
+        }
+
+        if let Some(screenshot) = self.screenshot.take() {
+            if let Some(mut path) = rfd::FileDialog::new().save_file() {
```

Comment:
Currently always defaults to "Untitled" (.png) but we could set a a different default filename if desired

Reply:
Maybe default to the name-of-the-profile.png?

Reply:
Or `Screenshot_YYYY-MM-DD-HH-MM-SS.png`, or something like that.

I agree a more informative default name is helpful.

Reply:
Giver there can be an arbitrary number of source locators I opted for something like @elliottslaughter's datetime suggestion https://github.com/StanfordLegion/prof-viewer/pull/43/commits/3be2ff1a1a31ca761cf39a02079716d8e3265f3b

<img width="424" alt="Screenshot 2024-01-11 at 12 40 52" src="https://github.com/StanfordLegion/prof-viewer/assets/1078448/3d4ab71e-efe0-45fe-b620-0228ed904ff3">